src/main/java/com/synopsys/integration/detect/lifecycle/boot/product/BlackDuckConnectivityChecker.java

```diff
@@ -70,6 +75,8 @@ public BlackDuckConnectivityResult determineConnectivity(BlackDuckServerConfig b
             );
         }
 
-        return BlackDuckConnectivityResult.success(blackDuckServicesFactory, blackDuckServerConfig);
+        BlackDuckConnectivityResult result =  BlackDuckConnectivityResult.success(blackDuckServicesFactory, blackDuckServerConfig);
```

Comment:
assignment seem unnecessary?

Reply:
that is needed to set the server version in the connectivity result for use later

Reply:
ah yes, for some reason I thought the return immediately followed the assignment but I see I had that wrong. Resolving.